backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/semantics/rdf/ColumnToRDF.java

```diff
@@ -16,10 +16,14 @@ public class ColumnToRDF {
 
   // todo: unit is missing (which would also be a sdmx-attribute:unitMeasure, typed as an
   // qb:AttributeProperty)
-  public static void describeColumns(ModelBuilder builder, Table table, String schemaContext)
-      throws Exception {
+  public static void describeColumns(
+      ModelBuilder builder, String columnName, Table table, String schemaContext) throws Exception {
     String tableContext = schemaContext + "/" + table.getName();
     for (Column c : table.getMetadata().getColumns()) {
+      // allow selecting one particular column by ignoring the rest
+      if (columnName != null && !c.getName().equals(columnName)) {
+        continue;
```

Comment:
you might want to learn the 'stream' interface. 

I am not sure it is shorter, but something like:

Column theOne = table.getMetadata().getColumns().
  .filter(c -> "c.getName().equals(columnName))
  .findAny();

Reply:
I'll keep it in mind!